### PR TITLE
Print mod source that called deprecated 'intllib.Getter'

### PR DESCRIPTION
**Pull request by AntumDeluge**
Helps in debugging which mods need updated.

### CODE REVIEW

**Comment by AntumDeluge** [comment]
I just realized that I can set log output level to 'info' or 'verbose' which will show what mod was loading when the warning occurred. So this PR is not necessary, but in my opinion, may still be helpful.

File: init.lua [hunk 1]
```diff
@@ -65,8 +65,10 @@ end
 
 
 function intllib.Getter(modname)
-	minetest.log("deprecated", "intllib.Getter is deprecated."
-			.."Please use intllib.make_gettext_pair instead.")
+	minetest.log("deprecated", "intllib.Getter (called from \""
+			..minetest.get_current_modname()
```
Comment (kaeza) [inline init.lua]: Suggestion: print source location instead:

```lua
local info = debug.getinfo(2)
local loc = info.short_src..":"..info.currentline
log("... (called from "..loc..") ...")
```

Comment (AntumDeluge) [reply init.lua]: I like it.

**Comment by kaeza** [comment]
d01a9c666f2d9f44a35562154ff86c7ce20e3e65

I reformatted the code a bit and added a few more checks (I'm a bit paranoid about `debug`).

Thanks!

**Pull request closed without merging**